### ISSUE TITLE
Solving the variable issue

### ISSUE THREAD

**Issue by phoenixanimations** [phoenixanimations/TheNuwasFile#76]
The game would be a lot easier to code with more variables on the character sheet. 

Cons:
+ Hard to read (for devs)
+ Names get increasingly abstract, I.e total damage and total damage multiplier.

Solution (current):
+ Grab data on the fly to create pseudo variables. 

Future solution:
+ Revamp character sheet. (Possibly not a solution)
+ Show all attributes in one spot

**Comment by RVNJ** [comment]
I'll come up with some ideas for sectioning off certain parts, or if it's possible we could limit certain variables to the class of the Equipment (preferred) with an override/debug for anything that doesn't use a standard Stat (a Weapon having HP for example). If we go with the latter, a Melee Weapon wouldn't show Magic or Archery stats but those stats could still be accessible via Inspector Debug(?*)

* I don't know if that would work but I would hope so, considering it shows Equipment equipped when hidden from the main Inspector

**Comment by phoenixanimations** [comment]
Yes we can do that and that's a good step. And yup it will work exactly how you predicted. However it will not solve the player stat wealth of info. 

Ideally I would like a solution that will make us not go "I don't want to add a new stat because it will clutter up the interface". As the accessibility of having the new stat on the character I think is too good to pass up. 

This is because "faking" stats (i.e. having a passive add up all the base stats) adds more headache to the file, and leaves the door open for glitches. You also make it really hard to modify anything that does "all" i.e. double healing (#69). 

But if there is a non-headache way to do this or something similar I'm up for it as an alternative. For now I don't think we'll make any changes, just something to think about for the future.

**Comment by phoenixanimations** [comment]
One thing we can try is a "hide" option. Somewhere else on the character sheet will give you an option to unhide the variable.

What do you think?
